README.md

```diff
@@ -202,6 +202,45 @@ full backup (f)
 6.  f f f f f f
 7.  f f f f f f f
 
+## Max backups limit behaviour
```

Comment:
The multiple full backup and single full backup sections above are no longer valid now that we're doing true incremental backups. Let me know if you'd like me to re-write the readme so that it fits the new behavior.

Reply:
Please take the tests and read me changes if possible. You should have write access to the branch